.github/workflows/zip.yml

```diff
@@ -9,8 +13,29 @@ jobs:
   build:
     runs-on: macOS-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
+    - name: Build
+      run: |
+        cd ZipBuilder
+        swift build -v
+
+  package:
+    needs: build
+    runs-on: macOS-latest
+    # Only run the `package` job for the cron job, not PRs.
+    if: github.event_name == 'schedule'
+    steps:
+    - uses: actions/checkout@v2
     - name: Setup Bundler
       run: ./scripts/setup_bundler.sh
     - name: ZipBuildingTest
-      run: ./scripts/build_zip.sh
+      run: |
+         mkdir -p zip_output_dir
+         sh -x scripts/build_zip.sh zip_output_dir
+
+    - uses: actions/upload-artifact@v1
+      with:
+        name: Firebase-actions-dir
+        # Zip the entire output directory since the builder adds subdirectories we don't know the
```

Comment:
Isn't this the directory the zip file is copied into?

Reply:
Not the case due to the Kokoro setup - the output dir is the root directory and artifacts get packaged in this structure:

```
# Zip is named `Firebase-latest-manual.zip` when no RC number is passed in.
<buildDir>/MAJOR_MINOR_PATCH/Firebase-latest-manual.zip
<buildDir>/carthage/MAJOR.MINOR.PATCH/latest-non-rc/<ALL_CARTHAGE_FILES>
```

This is used for the automated Kokoro setup, so unless we know the Firebase version we can't predict the full path of the output. I'm also fairly certain that regex doesn't work yet on the artifacts fetching, so we can't do something like `<buildDir>/*/Firebase-latest-manual.zip` currently.